src/components/Home.vue

```diff
@@ -206,10 +206,10 @@ export default {
                 if (!this.coldAddresses[i].hasOwnProperty('api')) {
                     let tempObj = {'protocol': 'v.systems', 'opc': 'account', 'address': i, 'api': 1, 'publicKey': this.coldAddresses[i]}
                     Vue.set(this.coldAddresses, i, JSON.parse(JSON.stringify(tempObj)))
-                    this.setUsrLocalStorage('coldAddresses', JSON.stringify(this.coldAddresses))
                 }
                 Vue.set(this.coldAddressesShow, i, this.coldAddresses[i].publicKey)
             }
+            this.setUsrLocalStorage('coldAddresses', JSON.stringify(this.coldAddresses))
```

Comment:
1. ```for (var i in this.coldAddresses)``` -> ```for (const addr in this.coldAddresses)```
2. Add a 'changing' flag variable to tag if ```coldAddresses``` in local storage requiring updates ( when ```if (!this.coldAddresses[i].hasOwnProperty('api'))``` is true)